sonata-project/admin-bundle/3.64/config/packages/sonata_admin.yaml

```diff
@@ -0,0 +1,12 @@
+sonata_admin:
+    title: 'Sonata Admin'
+    dashboard:
+        blocks:
+            - { type: sonata.admin.block.admin_list, position: left }
+    options:
+        legacy_twig_text_extension: false
+
+sonata_block:
```

Comment:
Is this a dependency of admin bundle?

Generally, recipes should not modify other bundles config. But sonata_block.blocks is a collection, right? Like framework.cache.pools

Reply:
>Is this a dependency of admin bundle?

Yes, this node is provided by "sonata-project/block-bundle"

>Generally, recipes should not modify other bundles config. But sonata_block.blocks is a collection, right? Like framework.cache.pools

I understand. Yes, `sonata_block.blocks` is a collection. Is there a way to prevent collisions with other bundles or previous configuration in a specific node? I mean, something like a merge or similar approach.

Reply:
No. There is no way (as I know) to prevent that. But adding things to collections is fine.